CONTRIBUTING.md

```diff
@@ -1,5 +1,8 @@
 # Contributing to Learn Open Source
 
-Guideline for contribution
-
 Thank you for investing your time in contributing to our project! 
+
+## Code of Conduct
+
+We take our open source community seriously and hold ourselves and other contributors to high standards of communication.
```

Comment:
it might be better to keep it in a single line :)

Reply:
others are seem to be fine to me!

Reply:
okay will do that!

Reply:
thanks!